platform-api/src/routers/forms.py

```
@@ -0,0 +1,125 @@
+            {"name": "Email", "value": body.email, "inline": True},
+            {"name": "Organization", "value": body.organization or "N/A", "inline": True},
+            {"name": "Message", "value": body.message[:1024]} # Discord limit is 1024
+        ],
+        "timestamp": datetime.now(timezone.utc).isoformat()
+    }
+    
+    try:
+        async with get_client() as client:
+            await client.post(webhook_url, json={"embeds": [embed]})
+    except Exception:
+        pass # Fire and forget, don't break the user's submission if Discord fails
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)